uwsgi.ini

```diff
@@ -2,3 +2,14 @@
 module = app
 callable = app
 uid = nobody
+# HTTP socket
+http-socket = :8080
+# Graceful shutdown on SIGTERM, see https://github.com/unbit/uwsgi/issues/849#issuecomment-118869386
+hook-master-start = unix_signal:15 gracefully_kill_them_all
+need-app = true
+die-on-term = true
+# For debugging and testing
+show-config = true
+# Default cheaper settings
+cheaper = 2
+processes = 16
```

Comment:
Probably they should not be defined in `uwsgi.ini`, leave them to be customized by user

Reply:
+1

Reply:
Fixed